### ISSUE TITLE
Bug regarding leap year 

### ISSUE THREAD

**Issue by eribul** [rOpenGov/sweidnumbr#26]
Problem with birth date on leap day seems to originate from some interaction of lubridate and pin_age?

Leap day (on a leap year):
````
> lubridate::leap_year(2000); lubridate::ymd("20000229"); pin_age("200002291234")
[1] TRUE
[1] "2000-02-29 UTC"
The age has been calculated at 2014-09-18.
Error in while (any(start + est * per < end)) est[start + est * per <  : 
  missing value where TRUE/FALSE needed
```

Almost the same numer but one day earlier (no leap day):
```
> lubridate::ymd("20000228"); pin_age("200002281234")
[1] "2000-02-28 UTC"
The age has been calculated at 2014-09-18.
[1] 14
```

**Comment by MansMeg** [comment]
Bug fixed in lubridate - need to uncomment the test case.

**Comment by MansMeg** [comment]
Bug fixed in lubridate 1.4 (but it is not yet on CRAN). Tests passed (but not on travis so tests are skipped on Travis)

**Comment by MansMeg** [comment]
Now fixed!
